### DISCUSSION TITLE
spare PCB

### DISCUSSION THREAD

**Discussion by peterdedecker83** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#103] [Ideas]
Hi

As I didn't have any experience with soldering and due to the ordering procedure, I have ordered 5 PCB. The first attempt was the right one, and it works like a charm. So now I have 4 PCBs left over. I'ld be happy to send one over snail mail to people who want to make their own spa smart as well, just at the shipping cost.

**Comment by OnurbHartmann** [comment]
Hi, I am interested in getting one. Can we get in touch via email. Thanks Bruno

**Reply by peterdedecker83** [reply]
Sure: you can mail me at spa at peterdedecker.eu

**Comment by peterdedecker83** [comment]
Sorry, I forgot to mention: I live in Belgium.

**Reply by Dellonb** [reply]
Hi, I am also interested in getting one. I will contact you via email, Regards Pawel

**Comment by Bishbosh02** [comment]
Hi I am just about to do the same but was going to add current sensor to my board as thats what I have on my old 4 wire esp32 board, but have now upgraded the pump to 6 wire so if you still have a spare board that I can use for testing ? I am in the UK

**Comment by GSzabados** [comment]
@peterdedecker83, do you have any spare PCBs left, I would be interested about one as well to Poland.

**Comment by jarisiv** [comment]
@peterdedecker83 Hi there! Any spares left? I am interested in one to be sent to Finland. Many thanks in advance!

**Comment by Bishbosh02** [comment]
I have 3 spare boards using a D1 Mini have also added power monitor to the analogue input  board needs a re-design  as the USB port clashes with the level converter chip :(  but works ok
![pcb](https://user-images.githubusercontent.com/9953147/144585347-bf3641e5-dd4c-43f9-8aa2-73f0678f0fc3.jpg)

**Reply by jarisiv** [reply]
Thanks, I actually found already a spare sender.
